src/style.css

```diff
@@ -0,0 +1,106 @@
+* {
+  margin: 0;
+}
+
+body {
+  background-color: #e5e5e5;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  background-color: #fff;
+}
+
+.product_img {
+  width: 100%;
+  margin-bottom: 40px;
+  height: 134px;
+}

```

Comment:
card width should de 160px. Also add display block, margin 0 auto 20px and your img will be in the center.